src/components/duende-og-image.tsx

```diff
@@ -1,6 +1,11 @@
 import React from "react";
 import type { RenderFunctionInput } from "astro-opengraph-images";
 const { twj } = await import("tw-to-css");
+import fs from "node:fs";
+import path from "node:path";
+
+const filePath = path.join(process.cwd(), "src/assets/duende-og-bg.png");
```

Comment:
I don't think Windows will like this. More path.join?

Reply:
I believe later versions of node do the handling of this properly. Let me check. Please hold.

Reply:
I've updated this line to just use the parts and let path.join find the correct path seperator. Should be cross-platform good now.